docs/tutorials/setup.md

```diff
@@ -86,13 +86,24 @@ libraries we will use. Again this depends on how you installed python, and wheth
 want to use `conda` or `pip`.
 
 Once you have a virtual environment setup and activated, install the libraries
-required with `pip install -r requirements.txt`. Now you are all ready to use our deployer
-scripts!
+required with `pip install -r requirements.txt -r dev-requirements.txt -e .`
+
+Now you are all ready to use our deployer scripts! Note if dependencies get
+added you will to run the installation again.
 
 Remember you need to *activate the environment* you installed these libraries into each
 time you use any of our scripts.
 
-## Step 3: Authenticate with Google Cloud to decrypt our secret files
+## Step 3: Setup the git pre-commit hooks
+
+In the root of the checked out `infrastructure` repo, with your choice of Python virtual
```

Comment:
`pre-commit` doesn't *require* to be installed with your virtual environment activated, and I personally think installing it outside of your virtual environment (via `brew` or `apt` or similar) is better. `pre-commit` also creates its own sub-virtual environments to install the configured pre-commit handlers and not touch the current env.

So I would suggest here just linking out to the installation instructions page and say 'install it'?

Reply:
fair enough. I just assumed hermetic environment for everything in the repo as I didn't dive into how pre-commit works